### ISSUE TITLE
nodeinfo: needs extension for protocol activitypub

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#5802]
### Expected behavior
As soon as PR #5794 is merged 'nodeinfo' should show ActivityPub as supported protocol.
Something like this, I guess:
````
$nodeinfo['protocols']['inbound'][] = 'activitypub';
$nodeinfo['protocols']['outbound'][] = 'activitypub';
````
### Friendica version you encountered the problem
2018.12-dev

### Friendica source (git, zip)
git

**Comment by AlfredSK** [comment]
Fixed by #7873

**Comment by AlfredSK** [comment]
Works! ❤️ Closing...
